CHANGELOG.md

```diff
@@ -22,6 +22,23 @@
 - **Breaking:** On Web, `set_cursor_position` and `set_cursor_grab` will now always return an error.
 - **Breaking:** `PixelDelta` scroll events now return a `PhysicalPosition`.
 - On NetBSD, fixed crash due to incorrect detection of the main thread.
+- On Wayland, add missing virtual keycodes.
+- On Wayland, implement proper `set_cursor_grab`.
+- On Wayland, the cursor will use similar icons if the requested one isn't available.
+- On Wayland, right clicking on decorations could show application menu.
+- On Wayland, fix repeat rate triggering multiple times from slow callback handler.
+- On Wayland, fix fullscreen state size tracking.
```

Comment:
What is "fullscreen state size"? I'd assume you just mean the fullscreen size across state changes?